sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -182,7 +179,7 @@ dif_rstmgr_result_t dif_rstmgr_software_reset(
   }
 
   mmio_region_t base_addr = handle->params.base_addr;
-  if (rstmgr_software_reset_locked(base_addr, peripheral)) {
+  if (rstmgr_software_reset_is_locked(base_addr, peripheral)) {
     return kDifRstmgrLocked;
```

Comment:
Would it make sense to define another enum for locked because this is the only function that returns it?

Reply:
I think you are right, have created the new `enum` to be used in this function.